tests/framework/Models/External/load_and_run_fmu.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+""" This example demonstrates how to use the FMU.get*() and FMU.set*() functions
+ to set custom input and control the simulation """
+
+from fmpy import read_model_description, extract, dump
+from fmpy.fmi2 import FMU2Slave
+from fmpy.util import plot_result, download_test_file
+import numpy as np
+import shutil
+import sys
+
+
+def simulateCustomInputFMU(fmuFilename,pathToRaven,show_plot=True):
+
+  # define the model name and simulation parameters
+
+  start_time = 0.0
+  stop_time = 3.0
+  step_size = 1.
```

Comment:
Some questions about this file:
1. do you need to add the copyright info in this file?
2. do we need to follow raven python standard for this file?

Reply:
1. Copyright added.  
2. I am not sure.  It is just to test that an fmu loads.